manual/manual/tutorials/coreexamples.etex

```diff
@@ -289,6 +289,91 @@ let rec insert x btree =
                 else Node(y, left, insert x right);;
 \end{caml_example}
 
+
+\subsection{Record and variant disambiguation}
+( This subsection can be skipped on the first reading )
+
+Astute readers may have wondered what happens when two or more record
+fields or constructors share the same name
+
+\begin{caml_example*}{toplevel}
+type first  = { x:int; y:int; z:int }
```

Comment:
You could use `first_record` for consistency with `first_variant`

Reply:
Good idea.